alacritty_terminal/Cargo.toml

```diff
@@ -35,9 +35,12 @@ winpty = { path = "../winpty" }
 mio-named-pipes = "0.1"
 miow = "0.3"
 dunce = "1.0"
-winapi = { version = "0.3.7", features = ["impl-default", "winuser", "synchapi", "roerrorapi", "winerror", "wincon", "wincontypes"]}
 widestring = "0.4"
 mio-anonymous-pipes = "0.1"
+[target.'cfg(windows)'.dependencies.winapi]
+version = "0.3.7"
+features = ["impl-default", "basetsd", "libloaderapi", "minwindef", "ntdef", "processthreadsapi",
+            "winbase", "wincon", "wincontypes", "winerror", "winnt", "winuser"]
```

Comment:
Why did you add new feature flags that previously weren't required at all?

I also think that if we need to break it over multiple lines anyways, we might as well stick with the old version of defining `winapi` as a single element instead of a nested field.

Reply:
They were in use before, but not specified in the manifest - out of sync.
One single line toml table, here comes the line wrap :metal: